src/backend/marsha/settings.py

```diff
@@ -172,6 +183,9 @@ class Base(Configuration):
 
     BYPASS_LTI_VERIFICATION = values.BooleanValue(False)
 
+    LRS_BASIC_AUTH = values.SecretValue()
+    LRS_URL = values.URLValue()
```

Comment:
As mentioned previously?
```suggestion
    LRS_BASE_URL = values.URLValue()
    LRS_STATEMENT_ENDPOINT = f"{LRS_BASE_URL}/statement"
```

Reply:
Hmm no because this is specific to the actual LRS so it doesn't solve what you point in your comment